Dockerfile

```diff
@@ -20,5 +20,6 @@ RUN apk update && apk upgrade && \
     apk add --no-cache bash curl wget
 
 COPY --from=0 /src/bin/* /bin/
+COPY --from=0 /src/configmap/* /bin/configmap/
```

Comment:
It is better to rename the dir to `config`? And I think config files should  be put in another directory